src/main/resources/templates/tat-trend.html

```diff
@@ -11,52 +11,49 @@
   <div layout:fragment="content">
     <h1 class="text-green-200 font-sarabun font-light text-32">TAT Trend Report</h1>
     <div id="trendReportContainer">
-      <div id="controlsContainer">
-        <div id="groupingButtons" style="display: flex; gap: 10px; margin-top: 10px;">
+      <div id="controlsContainer" class="flex items-center justify-between gap-2 mt-2">
+        <div id="groupingButtons" class="flex gap-2">
           <button id="weekButton" data-grouping="week"
-            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">Week</button>
+            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex items-center">Week</button>
           <button id="monthButton" data-grouping="month"
-            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">Month</button>
+            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex items-center">Month</button>
           <button id="quarterButton" data-grouping="fiscalQuarter"
-            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">Fiscal
+            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex items-center">Fiscal
             Quarter</button>
           <button id="yearButton" data-grouping="fiscalYear"
-            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">Fiscal
+            class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex items-center">Fiscal
             Year</button>
```

Comment:
Remove `flex` and `items-center` from the buttons. These are flexbox (container) properties, and the buttons are flex items, so these are doing nothing.